.gitlab-ci.yml

```diff
@@ -642,6 +642,9 @@ library:ci-stdlib2:
 library:ci-unimath:
   extends: .ci-template-flambda
 
+library:tlc:
```

Comment:
The naming should be `library:ci-tlc` and should go before unimath as to keep the current order.

Reply:
Thanks for the catch. (Sorry for such a fail.)